cloud-common/src/main/resources/application.yml

```diff
@@ -131,7 +131,12 @@ cb:
      internal: >
                 HDP 3.0 - Data Science: Apache Spark 2, Apache Zeppelin=hdp30-data-science-spark2;
```

Comment:
Any particular reason for not removing the old DS and EDW? Also, I think those are standalone ones, so should be renamed if kept.

Reply:
Temporarily leaving all the existing blueprints. The existing blueprints work - and some people have been using them. @doktoric  - any thoughts on this? Remove now, or remove later when these blueprints are updated again, and promoted to external from internal.